lib/ts/I_Layout.h

```diff
@@ -31,80 +31,93 @@
 #ifndef _I_Layout_h
 #define _I_Layout_h
 
+#include <string>
+#include "ts/string_view.h"
```

Comment:
It does in my Impl.. it does not in the std::impl ( as I have to play it safe with definition)